constants_test.py

```diff
@@ -24,8 +24,10 @@ class ConstantsTests(test_utils.GenericTestBase):
 
     def test_constants_file_is_existing(self):
         """Test if the constants file is existing."""
-        self.assertTrue(os.path.isfile(os.path.join(
-            'assets', 'constants.js')))
+        self.assertTrue(
```

Comment:
Ah. You don't need to break after every paren, that is hard to read. The original code was fine.

The thing you don't want to see is where the arguments in a paren get split across multiple lines, and the first argument is directly after the opening paren. Like:

```
    abc(def,
        ghi)
```

Reply:
Oh my bad, I will fix all of this and make changes to the test as well.

Reply:
The check should only work when there are arguments within paren i.e. in case of functions only?

Reply:
Yep, no need to break `abc()`.

Occasionally you might need to do:

```
    abc(
        very_long_arg)
```